### PR TITLE
Make Kitchensink table test case forward-compatible

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change makes the Markdown table test case in the Kitchensink page forward-compatible. Otherwise, without this change, it’s using the legacy/deprecated `SpecName` and `Spec2` macros — and is a legacy “manual” Specifications table (and we don’t have such tables in content any longer; rather, the tables are generated based on the `browser-compat` or `spec-urls` frontmatter metadata keys).

### CODE REVIEW

File: files/en-us/mdn/kitchensink/index.md [hunk 1]
```diff
@@ -113,9 +113,19 @@ _The formal syntax must be taken from the spec and added to the [MDN data reposi
...
-| -------------------------------------------------------------------------------------------- | -------------------------------- | ------- |
-| {{SpecName("HTML WHATWG", "#link-type-dns-prefetch", "dns-prefetch")}} | {{Spec2("HTML WHATWG")}} |         |
+### Markdown table
+
+| Constant name                | Value  | Description |
+| ---------------------------- | ------ | ----------- |
+| `QUERY_COUNTER_BITS_EXT`     | 0x8864 | The number of bits used to hold the query result for the given target. |
+| `CURRENT_QUERY_EXT`          | 0x8865 | The currently active query. |
+| `QUERY_RESULT_EXT`           | 0x8866 | The query result. |
+| `QUERY_RESULT_AVAILABLE_EXT` | 0x8867 | A Boolean indicating whether or not a query result is available. |
```
Comment (OnkarRuikar) [inline files/en-us/mdn/kitchensink/index.md:123]:
```suggestion
| `QUERY_RESULT_AVAILABLE_EXT` | 0x8867 | A Boolean indicating whether a query result is available. |
```

File: files/en-us/mdn/kitchensink/index.md [hunk 2]
```diff
@@ -113,9 +113,19 @@ _The formal syntax must be taken from the spec and added to the [MDN data reposi
...
+| `TIME_ELAPSED_EXT`           | 0x88BF | Elapsed time (in nanoseconds). |
+| `TIMESTAMP_EXT`              | 0x8E28 | The current time. |
+| `GPU_DISJOINT_EXT`           | 0x8FBB | A Boolean indicating whether or not the GPU performed any disjoint operation. |
```
Comment (OnkarRuikar) [inline files/en-us/mdn/kitchensink/index.md:126]:
```suggestion
| `GPU_DISJOINT_EXT`           | 0x8FBB | A Boolean indicating whether the GPU performed any disjoint operation. |
```

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
I wonder if we should add a `{{Specifications}}` entry on this page.

**Comment by sideshowbarker** [comment]
> I wonder if we should add a `{{Specifications}}` entry on this page.

Yeah that would seem useful. But I’m not at all familiar with what the Yari automation does with this page — how it consumes it, what it expects. Specifically, if we add something new to it, I wonder if there’s some corresponding change we’d need to make to the consuming code in Yari.

**Comment by teoli2003** [comment]
cc/ @caugner here.

**Comment by caugner** [comment]
@sideshowbarker @teoli2003 As long as the changes don't introduce any flaws, no change should be necessary to yari.

The corresponding tests are defined here:

https://github.com/mdn/yari/blob/789e5e366845b4789cead25a85d8ec4416bd9e17/testing/tests/developing.spec.js#L28-L77

**Comment by sideshowbarker** [comment]
> @sideshowbarker @teoli2003 As long as the changes don't introduce any flaws, no change should be necessary to yari.
> 
> The corresponding tests are defined here:
> 
> [mdn/yari@`789e5e3`/testing/tests/developing.spec.js#L28-L77](https://github.com/mdn/yari/blob/789e5e366845b4789cead25a85d8ec4416bd9e17/testing/tests/developing.spec.js#L28-L77)

Cool — thanks much

**Pull request merged**